scripts/socok8s/run.sh

```diff
@@ -32,6 +32,9 @@ git --no-pager show | sed 's/^/|@| /'
 export PREFIX=socok8s-ci-${pr_id}-${sha1}
 export OS_CLOUD=engcloud-cloud-ci
 export KEYNAME=engcloud-cloud-ci
+export INTERNAL_SUBNET=ccpci-subnet
+export SOCOK8S_DEVELOPER_MODE=True
```

Comment:
Are you sure about this? You think there won't be a need to run it in non-developer mode?

Reply:
No, I'm not sure about this, I copied it from the periodic job.

Reply:
It doesn't seem to be copied from https://github.com/SUSE-Cloud/automation/blob/202cc1b06c91b5fcf7be7e28bfe83f09b756774a/jenkins/ci.suse.de/pipelines/cloud-ccp-full-periodic.Jenkinsfile or https://github.com/SUSE-Cloud/automation/blob/202cc1b06c91b5fcf7be7e28bfe83f09b756774a/jenkins/ci.suse.de/ccp.yaml . Did you mean copied from somewhere else? It would be better if those two jobs didn't use code copies, but the same.

I'm not too familiar with the details, but I suppose either both should be changed or drop it here.